backend/controllers/activityList.js

```diff
@@ -51,6 +71,18 @@ const getActivityListFactory = elite2Api => {
 
     const offenderNumbersWithDuplicates = eventsAtLocation.map(event => event.offenderNo)
     const offenderNumbers = [...new Set(offenderNumbersWithDuplicates)]
+    const offenderBookingIds = await Promise.all(
```

Comment:
This is only needed for the attendance calls, maybe this needs to be placed behind a feature flag too?